packages/components/src/select-control/style.scss

```diff
@@ -1,16 +1,6 @@
 .components-select-control__input {
-	background: $white;
-	height: 36px;
-	line-height: 36px;
-	margin: 1px;
 	outline: 0;
-	width: 100%;
 	-webkit-tap-highlight-color: rgba(0, 0, 0, 0) !important;
-
-	@include break-medium() {
-		height: 28px;
-		line-height: 28px;
-	}
```

Comment:
These styles were already being overridden by the styles in the Emotion file.

Reply:
It looks like the `line-height` is not being overridden by the Emotion style — although maybe you deleted it since it's not necessary?

I also noticed that on `trunk` the `36px` line height makes the text look misaligned vertically:

![Screenshot 2023-02-09 at 15 37 12](https://user-images.githubusercontent.com/1083581/217843014-9363c77f-ec5d-473c-8e17-4933e90dc28d.jpg)

If we had to re-instate `line-height`, we should probably do it in Emotion styles, depending on the `size` prop?

Reply:
You're right, I removed it because it was unnecessary and also making it look wrong on mobile 👍